File: SuperMessenger/SuperMessenger/Models/MainPageModel.cs
using SuperMessenger.Models.EntityFramework;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SuperMessenger.Models
{
    public class MainPageModel
    {
        public Guid Id { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public Guid ImageId { get; set; }
        public int InvitationCount { get; set; }
        public int ApplicationCount { get; set; }
        public List<Country> Countries { get; set; }
        public List<SimpleGroupModel> Groups { get; set; }
    }
}

File: SuperMessenger/SuperMessenger/react-client/src/Models/MainPageModel.js
export default class MainPageModel { 
  constructor(id, email, firstName, lastName, imageName, invitationCount, applicationCount, countries, groups) {
    this.id = id;
    this.email = email;
    this.firstName = firstName;
    this.lastName = lastName;
    this.imageName = imageName;
    this.invitationCount = invitationCount;
    this.applicationCount = applicationCount;
    this.countries = countries;
    this.groups = groups;
  }
}